tasks/sf-sns-report/package.json

```diff
@@ -35,8 +35,8 @@
   "dependencies": {
     "@cumulus/api": "1.14.2",
     "@cumulus/cumulus-message-adapter-js": "^1.0.8",
-    "@cumulus/ingest": "1.14.2",
-    "lodash.get": "^4.4.2"
+    "lodash.get": "^4.4.2",
+    "lodash.isobject": "^3.0.2"
```

Comment:
Is this actually needed?     It doesn't see to be being required in the task, etc. 

```suggestion
```

Reply:
Good catch, that snuck back in when I merged master into this branch.